src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -1,31 +1,29 @@
 package org.broadinstitute.dsde.firecloud
 
-import scala.jdk.CollectionConverters._
-import com.typesafe.config.{ConfigFactory, ConfigObject}
-import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectiveUtils, NihWhitelist}
-import org.broadinstitute.dsde.rawls.model.{EntityQuery, SortDirections}
-import org.broadinstitute.dsde.workbench.model.{WorkbenchEmail, WorkbenchGroupName}
-
 import akka.http.scaladsl.model.Uri
 import akka.http.scaladsl.model.Uri.{Authority, Host, Query}
+import com.typesafe.config.{Config, ConfigFactory, ConfigObject}
+import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectiveUtils, NihWhitelist}
+import org.broadinstitute.dsde.rawls.model.{EntityQuery, SortDirections}
+import org.broadinstitute.dsde.workbench.model.WorkbenchGroupName
 
+import scala.jdk.CollectionConverters._
 import scala.util.Try
 
 object FireCloudConfig {
   private val config = ConfigFactory.load()
 
   object Auth {
     private val auth = config.getConfig("auth")
-    val googleClientId = auth.getString("googleClientId")
-    val googleSecretJson = auth.getString("googleSecretsJson")
```

Comment:
Interesting...what was googleSecretJson used for?  Was it just not used?

Reply:
It was not used at all :)

Reply:
right on